src/pal/tools/gen-buildsys-clang.sh

```diff
@@ -40,7 +42,11 @@ else
   exit 1
 fi
 
-desired_llvm_version=3.5
+if [ $OS = "FreeBSD" ]; then
+	desired_llvm_version=35
+else
+	desired_llvm_version=3.5
+fi
```

Comment:
I had that originally too, but felt it made that code to messy to break it up. But if you think it is better I will change it and try to make it look cleanly as possible.